src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -174,10 +182,18 @@ const LibraryAuthoringPage = ({ returnToLibrarySelection }: LibraryAuthoringPage
 
   const [searchParams] = useSearchParams();
 
-  if (isLoadingLibraryData) {
+  if (isLoadingLibraryData || isLoadingStudioHome) {
```

Comment:
I don't think we should block the page until "studio home data" is loaded. I think we should just always assume the libraries feature is enabled, and if it later loads and tells us that it's disabled, it's fine to show a flash of the UI before changing to an error message that "Libraries V2 is not enabled."

That will provide a faster experience for the 99% of users using this as intended. The number of users who ever see this error message should be extremely low.

Reply:
Fixed with [70b8db8](https://github.com/openedx/frontend-app-authoring/pull/1449/commits/70b8db80788e9be398b4705694863c915d2459d6).